src/commands/app/add/action.js

```diff
@@ -94,7 +94,7 @@ class AddActionCommand extends TemplatesCommand {
 
     switch (selection) {
       case 'orgActionTemplates': {
-        const supportedServiceCodes = new Set(orgSupportedServices.map(s => s.code))
+        const supportedServiceCodes = new Set(orgSupportedServices.map(s => `|${s.code}`)) // | symbol denotes an OR clause
```

Comment:
now I am curious 😄  what is this doing ?

Reply:
it's part of the Template Registry API, to designate that the search query item is an OR clause.
`apis=|foo,|bar` means give me templates with apis either (foo OR bar). If the `|` is omitted, it is by default an AND.
`apis=foo,bar` menas give me templates with apis (foo AND bar).

In this case we want all templates that support any of the services.

Reply:
Thanks :)